src/attrs.rs

```diff
@@ -101,6 +101,32 @@ impl FamilyOwned {
     }
 }
 
+/// Metrics, but implementing Eq and Hash using u32 representation of f32
+//TODO: what are the edge cases of this?
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub struct CacheMetrics {
+    pub font_size_bits: u32,
```

Comment:
I would be very reluctant to expose these fields directly. Getters / setters seem like the way to go here.

Reply:
I think just the into/from implementations for Metrics are enough.